implementations/rust/ockam/ockam_app/src/enroll/enroll_user.rs

```diff
@@ -35,7 +34,12 @@ pub async fn enroll_user(app: &AppHandle<Wry>) -> Result<()> {
     enroll_with_token(&context, app_state.options())
         .await
         .unwrap_or_else(|e| error!("{:?}", e));
-    app.trigger_global(app::events::SYSTEM_TRAY_ON_UPDATE, None);
+    // Update tray menu. Use block to release the write lock.
+    {
```

Comment:
Yeah you are right. In the last minute I did some changes to have the menu manipulation code inside the TrayMenu struct, instead of scattered around in the handlers. I didn't realize they all look the same now 😅  I'll reintroduce the event to handle this!